lib/scavenger_hunt/app/views/scavenger_hunt/answers/_form.html.slim

```diff
@@ -1,13 +1,13 @@
 = form_for @answer, url: game_clue_answer_path(@game, @clue) do |form|
   label.sr-only for="answer" Enter object title
-  = text_field_tag :answer, "", placeholder: "Enter object title"
-
+  = text_field_tag :answer, "", placeholder: "Enter object title", :required => true
```

Comment:
Please use modern hash syntax instead of the hash rocket where necessary (e.g. this should read `required: true` and not `:required => true`)